### DISCUSSION TITLE
cy.mount() get a page blank

### DISCUSSION THREAD

**Discussion by insistedMarin** [cypress-io/cypress#25110] [Component Testing]
![image](https://user-images.githubusercontent.com/50784434/207039025-d6538bfb-9464-4d4b-b95a-ab559c48b9f1.png)
I got a blank page with cy.mount(Component), someone knows how to fix this problem? thanks!

**Comment by lmiller1990** [comment]
Please share a minimal reproduction - impossible to say without more information.
